tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/TaskExecutorImpl.java

```diff
@@ -16,13 +16,19 @@
 import com.google.common.base.Optional;
 
 import org.apache.commons.compress.archivers.ArchiveException;
+
+import org.codehaus.plexus.util.FileUtils;
+
```

Comment:
Seems that we've been using styles for imports even within this project. Better keep it the way it's now to keep the style consistent within the file.